src/Mailer/Transport/AwsSesTransport.php

```diff
@@ -16,7 +20,7 @@ class AwsSesTransport extends AbstractTransport
      *
      * @var array
      */
-    protected $_defaultConfig = [
+    protected array $_defaultConfig = [
```

Comment:
This is to fix the fatal error on any page for project that uses this plugin.
For Bangdream, related error: https://redmine2.tyrellsys.com/issues/95208#note-21